Jenkinsfile

```diff
@@ -29,225 +23,154 @@ pipeline {
     }
     agent any
     stages {
-        stage('Checkout'){
+        stage('Checkout') {
             agent any
-            steps{
+            environment{ STAGE = "Checkout" }
+            steps {
                 deleteDir()
                 checkout scm
                 stash name: 'app'
-                sh "$SBT version"
+                sh "sbt version"
                 script {
                     version = '1.0.' + env.BUILD_NUMBER
```

Comment:
I'm not sure how useful this is. Because it's hardcoded and not taken from sbt they could fall out of sync. I would either remove and just use the build number, or we devise a solution for printing sbt properties (which I have tried, but wasnt successful with).

Reply:
sure, removed hardcoded version prefix